.gitignore

```diff
@@ -1,5 +1,3 @@
 config.yaml
 vendor
 store.bbolt
-rfd-server
-rfd-client
```

Comment:
IMO this should both stay.

if you do `go build ./cmd/rfd-client` or `go build ./cmd/rfd-server` it drops the binary here at root

Reply:
But adding those here wouldn't cause changes made in `./cmd/rfd-client/main.go` and `./cmd/rfd-server/main.go` to be ignored by git? :thinking: